### PR TITLE
Avoid creation of multiple machine requests for the same instance

### PR DESCRIPTION
**Pull request by prosif**
Before creating a new machine request, check for currently open machine requests based on the same instance and raise and exception if any are found.

Redone from PR #75 to be targeted to Kicking Kestrel

### CODE REVIEW

**Comment by arleigh16** [comment]
After a very long time I still cannot find a way to get off this mailing list.
 Arleigh Birchler "Wild Azalea Home" ** Sent from my Intergalactic Star Cruiser **

File: api/v2/views/machine_request.py [hunk 1]
```diff
@@ -27,6 +41,25 @@ def perform_create(self, serializer):
         new_owner_id=self.request.user.id
         parent_machine_id = serializer.validated_data['instance'].provider_machine.id
         status, _ = StatusType.objects.get_or_create(name="pending")
+        new_machine_provider = Provider.objects.filter(id=new_provider_id)
```
Comment (steve-gregory) [inline api/v2/views/machine_request.py]: One final note that this will all be changed over to UUID's soon.

**Comment by steve-gregory** [comment]
arleigh16 you need to 'Unwatch' and 'Unstar' the project here: https://github.com/iPlantCollaborativeOpenSource/atmosphere

**Comment by steve-gregory** [comment]
@prosif LG2M. :shipit:

**Pull request merged**